src/components/Project/ProjectsAPI.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com",
+});
+
+export default async function GetProjects() {
+  try {
+    let result = await (await instance.get("/users")).data;
+    return result;
+  } catch (error) {
+    console.log(error);
```

Comment:
remove this console.log please
Handle the error and do or return something to let your component know that the API call failed. You could also handle the case when you get an empty list of projects.